ingest/ingest_pipeline.py

```diff
@@ -374,11 +355,11 @@ def create_parser():
     )
 
     parser.add_argument(
-        "--study-accession",
+        "--study-file-id",
```

Comment:
It seems your renamings got mistakenly swapped.  I suspect you intended:
* `--study-accession` -> `--study-id`
* `--file-id` -> `--study-file-id`

but instead did:
* `--study-accession` -> `--study-file-id`
* `--file-id` -> `--study-id`

This suggestion and the next one fix that.

```suggestion
        "--study-id",
```